src/routes/safe/components/Transactions/helpers/TxParametersDetail/index.tsx

```diff
@@ -66,9 +86,12 @@ export const TxParametersDetail = ({
   const safeNonceNumber = parseInt(safeNonce, 10)
   const lastQueuedTxNonce = useSelector(getLastTxNonce)
   const showSafeTxGas = useSafeTxGas()
+  const storedTx = useSelector((state: AppReduxState) => getTransactionsByNonce(state, safeNonceNumber || 0))
```

Comment:
Could this return wrong data if we use a 0 nonce as fallback?

Reply:
It would fallback to 0 if there is no nonce set, e.g. it is the first transaction on the Safe.